projects/observability/src/shared/dashboard/widgets/topology/edge/curved/entity-edge-curve-renderer.scss

```diff
@@ -18,7 +18,7 @@
       stroke-width: 2;
       stroke-linecap: round;
       stroke-linejoin: round;
-      stroke: currentColor;
+      stroke: $gray-3;
```

Comment:
Should this be changing?

Reply:
This I can look into, I think it was needed, since currentColor was not working.

Reply:
Please take a look